microservices/netinfra_utils/workers/vll_service_worker.py

```diff
@@ -236,6 +239,17 @@ def service_delete_vll_task(task, commit_type):
     device_1 = service.devices[0]
     device_2 = service.devices[1]
 
+    service_to_delete = ServiceDeletion.parse_from_task(task)
+    if service_to_delete.devices[0].interface_reset:
```

Comment:
even though there are just 2 devices in a vll service, foreach loop would make sense here.

Also, what if I delete VLL and only reset ifc on one of the devices ? would this fail ?

Reply:
ok I rework it to loops, and no it won't fail, because interface_reset is set to False by default